README.md

```diff
@@ -1,3 +1,121 @@
-# Sephora Tester Tracker
+# Tester Tracker
 
-Testers Tracker is an inventory tracking app that allows Sephora team members to manage their store’s backstage product testers.
+## Description
+Tester Tracker is an inventory management tool that allows Sephora team members to manage their store’s backstage product testers.
+
+### Disclaimer
+**This application is not affiliated with, endorsed by, or officially connected to Sephora in any way.** It is solely a personal project developed as a playground environment for educational and experimental purposes. The app does not represent real employees, processes, or operations at Sephora. Any references to Sephora, its roles, or its products are purely fictional and for illustrative purposes only. **If you have any concerns regarding the use of brand names or references, feel free to reach out via the contact information below.** 
+
+## Table of Contents
+- [Features](#features)
+- [Installation](#installation)
+- [Usage](#usage)
+- [Entity Relationship Diagram](#entity-relationship-diagram)
+- [Troubleshooting](#troubleshooting)
+- [Contributing](#contributing)
+- [Contact](#contact)
+
+## Features
+- Role-specific views for Operations Associates, Admins, Beauty Advisors, Leads, and Managers, each with access to relevant features for their responsibilities.
+- Search for products by name or SKU.
+- Create and manage tester inventory.
+- Trash and approve testers.
+
+## Installation
+
+### Prerequisites
+- Ruby 3.x
+- Rails 7.x
+- PostgreSQL
+
+1. Clone the repository:
+`git clone https://github.com/dgarcia13-99/tester-tracker-sephora`
+
+2. Navigate to the project directory:
+`cd tester-tracker-sephora`
+
+3. Install the required gems:
+`bundle install`
+
+4. Set up the database:
+`rake sample_data`
+
+5. Start the Rails server:
+`bin/dev`
+
```

Comment:
also, are there any configurations to your project? I don't believe you are using any API's at the moment, but maybe put something in here saying like under a new configuration section saying like "no configurations at the moment, if you would like to implement something of that nature,  please refer to the `rails credentials`..." and then link to that